controller/hardware/sensor/hardware-sensor.hpp

```diff
@@ -4,9 +4,9 @@ class Sensor
 {
     public:
 
-        virtual void init(void) = 0;
+        virtual void init() = 0;
 
-        virtual void update(void) = 0;
+        virtual void update() = 0;
 
-        virtual float fetch_sample(void) = 0;
+        virtual float fetch_sample() = 0;
```

Comment:
Mmmm, I don't really like this virtual generic method. I think we should have each class implement its own get function.